mindsdb_native/libs/phases/data_cleaner/data_cleaner.py

```diff
@@ -3,28 +3,32 @@
 
 
 class DataCleaner(BaseModule):
-    def _cleanup_w_missing_targets(self, df):
+    def get_empty_columns(self, df):
+        empty_columns = []
+        for col_name in df.columns.values:
+            if len(df[col_name].dropna()) < 1:
+                empty_columns.append(col_name)
+                self.log.warning(f'Column "{col_name}" is empty ! We\'ll go ahead and ignore it, please make sure you gave mindsdb the correct data.')
+
+        return empty_columns
+
+    def remove_missing_targets(self, df):
```

Comment:
"private" methods should use `_` in front, at least that's the style we went with